leadership.html

```diff
@@ -332,83 +326,76 @@ <h1>رهبری</h1>
-                the company goals
+                <span class="highlight underline">انگیزه دادن به دیگران و هماهنگ‌سازی</span>
+                اطمینان حاصل می‌کند که <span class="highlight">اعضای تیم</span> استراتژی و مبانی اصلی شرکت را درک کرده و با اهداف شرکت همسو هستند. 
               </li>
               <li>
-                <span class="highlight underline">Expectation Management</span>
-                Anticipates and communicates blockers and delays, and cost ballooning within their team’s projects,
-                before they require escalation with suggested solutions.
-                <br />Ensures expectations with their team and external stakeholders are clarified between all parties
-                involved.
+                <span class="highlight underline">مدیریت انتظارات</span>
+                موانع، تاخیرها و افزایش هزینه‌ها را در پروژه‌های تیم خود پیش‌بینی کرده و در موردشان صحبت می‌کند و قبل از اینکه مشکلات پیچیده و حاد شوند، برای آن‌ها راه حل‌هایی پیشنهاد می‌دهد.
+                <br />اطمینان حاصل می کند که انتظارات با تیم و ذینفعان خارجی بین همه طرف‌های درگیر شفاف شده است.
               </li>
             </ul>
           </div>
-          <a class="link" id="show-teamlead-all" onclick="showAll('teamlead')">+ Show All Competencies</a>
-          <a class="link hide" id="hide-teamlead-all" onclick="hideAll('teamlead')">+ Hide Other Competencies</a>
+          <a class="link" id="show-teamlead-all" onclick="showAll('teamlead')">+ نمایش تمام شایستگی‌ها</a>
+          <a class="link hide" id="hide-teamlead-all" onclick="hideAll('teamlead')">+ پنهان کردن شایستگی‌های دیگر</a>
         </div>
 
         <div class="right col">
-          <div class=title>Examples</div>
-          <a class="show link" id="teamlead-show" onclick="showExamples('teamlead')">+ show examples</a>
-          <a class="hide link" id="teamlead-hide" onclick="hideExamples('teamlead')">- hide examples</a>
+          <div class=title>مثال‌ها</div>
+          <a class="show link" id="teamlead-show" onclick="showExamples('teamlead')">+ نمایش مثال‌ها</a>
+          <a class="hide link" id="teamlead-hide" onclick="hideExamples('teamlead')">- پنهان کردن مثال‌ها</a>
           <ul class="hide" id="teamlead-examples">
-            <li>Discusses paths, and creates plans for personal and professional growth</li>
-            <li>Works quickly to align the team when they need clarification on goals</li>
-            <li>Facilitates and runs meetings on technical planning for a project</li>
+            <li>در مورد مسیرها بحث می کند و برای رشد شخصی و حرفه ای برنامه ریزی می کند</li>
+            <li>به سرعت کار می‌کند تا تیم را در زمانی که نیاز به شفاف سازی در مورد اهداف دارند، هماهنگ کند</li>
+            <li>جلسات برنامه ریزی فنی یک پروژه را تسهیل و اجرا می کند</li>
             <li>Lead team discussions and research to identify risks. Front loads these risk in projects and
               communicates these early to stakeholders</li>
```

Comment:
I wonder what does `front loads` mean in this sentence

Reply:
I'm not sure, but she probably means to raise the risk first. nah?